demos/demo_runner/iot_demo_freertos.c

```diff
@@ -283,6 +283,7 @@ static int _initialize( demoContext_t * pContext )
  */
 static void _cleanup( void )
 {
+    AwsIotNetworkManager_DisableNetwork( configENABLED_NETWORKS );
```

Comment:
Should MQTT unsubscribe happen before disabling network? 

In setup -- we start network first, then MQTT connect, then subscribe. In tear down, I think we should do things in the exact opposite order.

Reply:
I thought about this and Ravi told me the order doesn't matter here. 
@ravibhagavandas do you have any comment?

Reply:
Subscriptions are stored separately from network enable/disable context. So it shouldn't matter if you remove subscriptions  later